changelog.md

```diff
@@ -73,8 +75,3 @@
 
 ## Tool changes
 
-- The rst parser now supports markdown table syntax.
```

Comment:
I delete these contents wrongly, please add them back

Reply:
@xflywind why not push a commit to this PR to do that?

Reply:
How could I do that?

Reply:
either:
* make a PR against this PR (select authors's fork/branch when you make your PR)
* use the github review suggestion which author can commit
* (since it's mostly orthogonal to this PR) do a separate PR, probably simplest

Reply:
Hes admin on my fork now.